### DISCUSSION TITLE
SlidingWindowSplitter

### DISCUSSION THREAD

**Discussion by TDL77** [pycaret/pycaret#3432] [Q&A]
Please tell me how to implement this piece of code in pycaret.__version__ 3.0.0  ?
from here 
https://github.com/pycaret/pycaret/discussions/1761
----------------------------------------------------------------------------
from pycaret.time_series import TSForecastingExperiment
from pycaret.internal.pycaret_experiment.time_series_experiment import TimeSeriesExperiment
exp = TimeSeriesExperiment()
my_custom_cv = SlidingWindowSplitter(fh=np.arange(1, 25), window_length=48, step_length=6)
exp.setup(data=y, session_id=42, fold_strategy=my_custom_cv)

exp.plot_model(plot="cv")
------------------------------------------------------------------------
RESULT:
No module named 'pycaret.internal.pycaret_experiment.time_series_experiment'
OR
name 'SlidingWindowSplitter' is not defined

**Comment by ngupta23** [comment]
This is a very old piece of code. The imports have changed. Refer to the tutorial folder for the right way to import this. 

Also if you can tell us where you got this piece of code from, we can update it. Thanks.
